packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md

```diff
@@ -1,26 +1,22 @@
 ---
-title: Identity engine
+title: Identity Engine
 meta:
   - name: description
-    content: Identity engine offers customizable building blocks that can support dynamic, app-based user journeys. Find out more about Identity engine, why you would use it, and how to upgrade your org.
+    content: Identity Engine offers customizable building blocks that can support dynamic, app-based user journeys. Find out more about Identity Engine, why you would use it, and how to upgrade your org.
 ---
 
-# Identity engine
+# Identity Engine
 
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-## About Identity engine
+## About Identity Engine
 
-Identity engine allows organizations to customize their Okta cloud components to satisfy an unlimited number of identity use cases. Instead of a reliance on pre-defined behavior for authentication, authorization, and enrollment, Identity engine offers customizable building blocks that can support dynamic, app-based user journeys.
+The Okta Identity Engine is a platform service that powers flexible and customizable access experiences, which allows enterprises to build access experiences tailored to their organizational needs. Just one of the many goals of Okta Identity Engine is that customers are no longer bound to any one way of identifying, authorizing, enrolling, and issuing access to users. Instead, they can customize and extend each of these steps, which delivers more flexible and customizable access experiences, including passwordless sign-in, improved per app experiences, enhanced context for secure authentication, and more.
```

Comment:
Pure suggestions here. In first sentence, I found the two "access experiences" a little clunky so close together. 

`The Okta Identity Engine is a platform service that powers flexible and customizable access experiences, which allows enterprises to tailor these experiences based on their organizational needs.`

And second sentence, maybe move away from "is that" construction:

`One of the many goals of Okta Identity Engine is to no longer bind customers to any one way of identifying, authorizing ...etc.`